docker-hub/roles-and-permissions.md

```diff
@@ -0,0 +1,72 @@
+---
+description: Use roles in your organization to control who has access to content, registry, and organization management permissions.
+keywords: members, teams, organization, company, roles, access
+title: Roles and permissions
+---
+
+Organization and company owners can assign roles to individuals giving them different permissions in the organization. This section is for owners who want to learn about the defined roles and their permission scopes.
+
+## Roles
+
+When you invite users to your organization, you assign a role. A role is a collection of permissions. Roles define access to perform actions like creating repositories, pulling images, publishing extensions, or managing repositories.
```

Comment:
> Roles define access to perform actions like creating repositories, pulling images, publishing extensions, or managing repositories.
All the examples are repos related. I would add more repos not-related permissions like creating a team, changing organization settings, etc.

Reply:
Wil update!